### ISSUE TITLE
Application can not open files

### ISSUE THREAD

**Issue by da2x** [leonardschardijn/Chirurgien#5]
```
$ chirurgien ~/some.file
GLib-GIO-CRITICAL: This application can not open files
```

Also, the `.desktop` file needs an `MimeType=all/allfiles;` entry.

**Comment by leonardschardijn** [comment]
Sure, opening files from the command line can be easily added. I'll include it in the next release.

**Comment by leonardschardijn** [comment]
Included in the v2.1 release
